src/config/language.c

```diff
@@ -37,10 +41,27 @@ void CONFIG_ReadLang(u8 idx) {(void)idx;}
 void CONFIG_EnableLanguage(int state) {(void)state;}
 #else
 static u16 fnv_16_str(const char *str);
-static char strings[8192];
 static u16 table_size;
-#define MAX_STRINGS 475
+static unsigned fix_crlf(char *str);
+
+#ifndef MAX_LINE
 #define MAX_LINE 300
+#endif
+
```

Comment:
check_string_size.pl won't understand this.  probably need to try to extract the length from the object file

Reply:
object file may not have it. maybe map file which contains variables' size.

Reply:
Removed so MAX_LINE is constant now